### DISCUSSION TITLE
Homepage member panel shows last visit instead of last login

### DISCUSSION THREAD

**Discussion by robinsowell** [ExpressionEngine/ExpressionEngine#3203] [Q&A]
Had a user notice that on the homepage, the date for the member is showing what is basically the last visit date, which doesn't mesh with the last login date in the logs.  They would expect it to match the log date, which does feel more accurate.

That said, I'm not sure that's supposed to be limited to cp visits or not.  But worth taking a look.  Last visit date can be a bit confusing.

<img width="1286" alt="Screenshot 2023-04-03 at 3 11 24 PM" src="https://user-images.githubusercontent.com/1181219/229604959-161a3f05-5f27-444d-8989-e215c3f22ef7.png">

**Comment by ic360-com** [comment]
I find this one annoying sometimes too.
As an Admin... I sometimes would like to see when certain users were last using the CP.

**Comment by shbchk** [comment]
I understand "Last visit" as a date when a user was last active on the website. It turns out it's not! 

At the same time, we have "Last activity" which is more valuable.

![photo_2023-08-09_21-57-13](https://github.com/ExpressionEngine/ExpressionEngine/assets/12234507/bbea7506-0a2e-44b3-85b3-3b6d28e74f9b)
